addons/mail/tests/test_mail_render.py

```diff
@@ -147,44 +129,46 @@ def test_evaluation_context(self):
             '<span>Context Custom Context Value, value Custom Render Value</span>'
         ]
         for src, expected in zip(srces, results):
-            for engine in ['jinja']:
+            for engine in ['small_qweb']:
                 result = MailRenderMixin.with_context(**custom_ctx)._render_template(
                     src, partner._name, partner.ids,
                     engine=engine, add_context=add_context
                 )[partner.id]
                 self.assertEqual(expected, result)
 
     @users('employee')
-    def test_render_mail_template_jinja(self):
-        template = self.env['mail.template'].browse(self.test_template_jinja.ids)
+    def test_render_mail_template_small_qweb(self):
+        template = self.env['mail.template'].browse(self.test_template_small_qweb.ids)
         partner = self.env['res.partner'].browse(self.render_object.ids)
         for fname, expected in zip(['subject', 'body_html'], self.base_rendered):
-            rendered = template._render_field(
+            rendered = str(template._render_field(
```

Comment:
why the `str`?

Reply:
It had mattered at some point but indeed, it does not matter anymore. I remove it.